client/my-sites/sharing/connections/account-dialog.jsx

```diff
@@ -113,12 +113,21 @@ class AccountDialog extends Component {
 		const connectedAccounts = this.getAccountsByConnectedStatus( true );
 
 		if ( connectedAccounts.length ) {
+			const hasConflictingAccounts = this.isSelectedAccountConflicting();
+
 			return (
 				<div className="account-dialog__connected-accounts">
 					<h3 className="account-dialog__connected-accounts-heading">{ this.props.translate( 'Connected' ) }</h3>
 					<ul className="account-dialog__accounts">
 						{ this.getAccountElements( connectedAccounts ) }
 					</ul>
+					{ hasConflictingAccounts &&
+					<Notice
+						status="is-warning"
+						icon="notice"
+						text={ this.props.translate( 'The marked connection will be replaced with your selection.' ) }
+						isCompact />
```

Comment:
Would we have needed to apply `showDismiss={ false }` since the default is true and previously we'd disabled it? Or are styles hiding the dismiss?